### ISSUE TITLE
[BUG] SteamVR borks GP102 HDMI Audio Controller profiles

### ISSUE THREAD

**Issue by Lyle-Tafoya** [ValveSoftware/SteamVR-for-Linux#302]
**Describe the bug**
My graphics card (GTX 1080ti) has 4 ports that audio can be outputted from. It has 1 HDMI port and 3 DisplayPort ports. Since the Vive requires the single HDMI port, I use DisplayPort for my normal audio/video. Specifically, I use a DisplayPort->HDMI cable since my devices require HDMI-in (ie. my TV or audio receiver). If I open "pavucontrol" or "plasma-pa" (pulseaudio audio management applications), this shows up as "GP102 HDMI Audio Controller" with numbered profiles HDMI0 through HDMI4 depending on which port on my graphics card I am using. The single HDMI port is associated with the profile "HDMI2". When I start SteamVR, it immediately switches to the HDMI2 profile. I do not use my Vive's audio regardless, so I consider this to be undesirable behavior. However, that's not the real issue I am reporting. The real issue is that the HDMI profile I use for my primary audio becomes unavailable/unusable. What's worse is that closing SteamVR does not re-enable the original profile I was using, leaving me completely without the ability to output audio via the GP102 HDMI Audio Controller. Even restarting pulseaudio does not bring back the ability to select the desired profile. Restarting my computer seems to restore things back to normal. It's worth noting that USB and AUX audio devices are unaffected by this issue.

**To Reproduce**
1. Plug AV device (ie. TV) into a DisplayPort port on a GTX 1080ti
2. Plug OG Vive into HDMI port on the same GTX 1080ti
3. Set GP102 HDMI Audio Controller profile associated with your AV device as the active profile
4. Start SteamVR
5. Attempt to switch back to original audio profile (this doesn't work)
6. Close SteamVR
7. Attempt to switch back to the relevant audio profile (this doesn't work)

**Expected behavior**
SteamVR should not bork my GP102 HDMI Audio Controller profiles. I should be able to use any such profiles while using my Vive as well as after SteamVR has been closed.

**System Information (please complete the following information):**
- Distribution: Arch Linux
- SteamVR version: beta 1.10.15 (also reproducible with stable build)
- Steam client version: [build number or date] Feb 20 2020, at 19:27
- Opted into Steam client beta?: No
- Graphics driver version:  Nvidia 440.59
- Gist for SteamVR System Information: https://gist.github.com/Lyle-Tafoya/86ed56193b6941c3db6e428960f2f127

**Additional context**
This problem has been ongoing for about 1.5-2 years. It did not used to be a problem before then.

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by Lyle-Tafoya** [comment]
I found a couple ways to workaround this issue. Instead of restarting my computer I can restart my display manager (sddm in this case) or I can unplug the DP cable from my graphics card and plug it back in.

**Comment by rnd-ash** [comment]
This is more of a KDE/sddm issue than something wrong with steamvr.

I used to use gnome and everything was perfect with VR audio wise, installed KDE and experienced utter hell with sound system selection, it got to a point where each time I plugged my Vive into my PC (GTX1060) I had to restart to get it to see the hdmi Audio output of the Vive 

One fix that I found is to simply force load the sound outputs of the GPU in pulse audio config file. Unfortunately this means you always see the outputs, even if nothing is plugged into them.
